pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/SimpleMinionClusterIntegrationTest.java

```diff
@@ -89,22 +88,35 @@ public void setUp() throws Exception {
 
   @Test
   public void testStopAndResumeTaskQueue() throws Exception {
+    // Hold the task
+    TestTaskExecutor.HOLD.set(true);
+
     // Should create the task queues and generate a task
-    Assert.assertEquals(_taskManager.scheduleTasks().size(), 1);
-    Assert.assertEquals(_helixTaskResourceManager.getTaskQueues().size(), 2);
+    Assert.assertTrue(_taskManager.scheduleTasks().containsKey(TestTaskGenerator.TASK_TYPE));
+    Assert.assertTrue(_helixTaskResourceManager.getTaskQueues()
+        .contains(PinotHelixTaskResourceManager.getHelixJobQueueName(TestTaskGenerator.TASK_TYPE)));
 
     // Should generate one more task
```

Comment:
can u also fix the comment if it is not applicable? We are only testing for presence of task type, maybe 1, maybe many, right?

Reply:
The scheduleTasks() will generate at most one task under each task type, so we just need to check the key.